client/src/pages/admin/pages/plugins.tsx

```diff
@@ -1,11 +1,248 @@
-import React from 'react';
+import { PluginDefinition } from '@/api/Models';
```

Comment:
I think you need to split this into the Add/Create/List components

Reply:
I think there is no need to create one file per component  but can split the create and add into their own components but I would keep them in this same file for now until we think there is reason to split I would say. 

https://www.youtube.com/watch?v=nYNn_sNcBbM

Reply:
yep much more readable thanks